recipes/mpart/LICENSE.txt

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2015-2022, conda-forge
```

Comment:
The license needs to be the one for the package, not the conda-forge one for recipes.

Reply:
Sorry about that.  The license file has been updated.  Thanks!